pkg/lib/cockpit-components-empty-state.jsx

```diff
@@ -54,7 +54,7 @@ EmptyStatePanel.propTypes = {
     icon: PropTypes.oneOfType([PropTypes.string, PropTypes.func]),
     title: PropTypes.string,
     paragraph: PropTypes.string,
-    action: PropTypes.string,
+    action: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
```

Comment:
`string` is a valid `node`, so just checking the latter is enough.

Also, please fix the typo in the commit message.